lib/modules/manager/scalafmt/extract.ts

```diff
@@ -4,7 +4,7 @@ import * as semverVersioning from '../../versioning/semver';
 import type { PackageDependency, PackageFileContent } from '../types';
 
 const scalafmtVersionRegex = regEx(
-  'version *= *(?<version>\\d+\\.\\d+\\.\\d+)',
+  'version *= *"?(?<version>\\d+\\.\\d+\\.\\d+)"?',
```

Comment:
Could also single quotes be used here?

Reply:
I tried as I was not sure, single quotes are not supported, they raise an error in scalafmt.